### ISSUE TITLE
Create FastAPI framework for Kaiko

### ISSUE THREAD

**Issue by CamiloPosso** [microbiomedata/kaiko_metaproteome#6]
Create FastAPI functions to help with deployment of Kaiko into existing workflows.

Details to follow as we get closer to finishing Kaiko, add them here.

**Comment by ssarrafan** [comment]
This is more than a year old. Any plans to still work on this? @CamiloPosso @pdpiehowski

**Comment by CamiloPosso** [comment]
I don't think we will work on this, for now I will close it. Thanks @ssarrafan
